nebula-logger/plugins/opentelemetry/plugin/opentelemetry/classes/OpenTelemetryLoggerPlugin.cls

```diff
@@ -0,0 +1,461 @@
+//------------------------------------------------------------------------------------------------//
+// This file is part of the Nebula Logger project, released under the MIT License.                //
+// See LICENSE file or go to https://github.com/jongpie/NebulaLogger for full license details.    //
+//------------------------------------------------------------------------------------------------//
+
+/**
+ * @group Plugins
+ * @description Optional plugin that integrates with OpenTelemetry to send log records using the OTLP format
+ */
+@SuppressWarnings('PMD.ApexCRUDViolation, PMD.CyclomaticComplexity, PMD.ExcessivePublicCount')
+public without sharing class OpenTelemetryLoggerPlugin implements LoggerPlugin.Triggerable, System.Queueable, Database.AllowsCallouts {
```

Comment:
this should also implement `System.Finalizer` so that unhandled errors can be logged